frontend/__tests__/KanbanBoard.test.js

```diff
@@ -2,24 +2,43 @@ import { mount } from '@vue/test-utils'
 import { VueDraggableNext } from 'vue-draggable-next'
 import KanbanBoard from '../src/components/kanban/KanbanBoard.vue'
 import JobCard from '../src/components/kanban/JobCard.vue'
-import { expect, beforeEach, describe, it, vi } from 'vitest'
+import { expect, beforeEach, describe, it } from 'vitest'
 import testColumnMapping from './test_data/test_column_mapping.json'
 import testJobs from './test_data/test_jobs.json'
 
 describe('KanbanBoard', () => {
   let wrapper
-  let showDetailModal = vi.fn()
 
   beforeEach(async () => {
     wrapper = mount(KanbanBoard, {
       props: {
-        jobs: testJobs,
+        jobs: getJobsByColumn(),
         columns: testColumnMapping,
-        showDetailModal,
       },
     })
   })
 
+  // Processing in JobTrackingView
+  function getJobsByColumn() {
+    let jobsByColumn = []
+
+    testJobs.forEach((job) => {
+      if (!jobsByColumn[job.columnId]) {
+        jobsByColumn[job.columnId] = []
+      }
+      jobsByColumn[job.columnId].push(job)
+    })
+
+    testColumnMapping.forEach((column) => {
+      if (jobsByColumn[column.id].length > 0) {
+        jobsByColumn[column.id] = jobsByColumn[column.id].sort(
+          (a, b) => a.id - b.id,
+        )
+      }
+    })
+    return jobsByColumn
+  }
+
```

Comment:
Not a huge fan of using a function like this to process test data. Instead can you paste in what this function will output with the test data as a string/json object and use that instead, in other words: hardcode the result?